### PR TITLE
Bugfix/markdown

### PR DESCRIPTION
**Pull request by butlerx**
markdown formatting errors #1 from the original repo

### CODE REVIEW

File: docs/network/mainaddressspace.md [hunk 1]
```diff
@@ -95,23 +85,22 @@ Currently not in use.
...
-Yellow: Management VLAN\\
-Green: RBVM Subnet crossover\\
-
+Red: External VLAN
+Blue: Internal VLAN
+Yellow: Management VLAN
+Green: RBVM Subnet crossover
 
-{{:network-map.png|}}
+![Network Diagram](/img/network-diagram171904.png)
```
Comment (VoyTechnology) [inline docs/network/mainaddressspace.md]: Related to discussion on #61

File: docs/plans/project-ideas.md [hunk 2]
```diff
@@ -1,66 +1,61 @@
...
-     * Hack something together to turn them into backups
-       * could you use some sort of VM for this? give it access to the device?
-       * you could mount them rw on a mount point that was chmoded 700, and then remount them somewhere else ro that users can actually read. Hack, but it'll work.
-     * Mount and offer to members as slow-storage for large files
+## Larger Projects
 
-*  Post Install Script for ubuntu to automaticaly fix security and other bugs, may also automatically install certain packages see [New Installs Page](/procedures/newinstalls) for list of common bugs to be fixed
- 1. -something like 'apt-get install redbrick-server'? The only thing this doesn't do, iirc, is change the config on sshd to prevent root logins
+* Web hey/huh interface
+  * I may look into this over the summer - creadak
```
Comment (VoyTechnology) [inline docs/plans/project-ideas.md]: `>` would be more appropiate?

File: docs/plans/project-ideas.md [hunk 3]
```diff
@@ -1,66 +1,61 @@
...
+* Improvement of web boards interface to appeal to newbies (David)
+  * If you're doing this, you should hang around boards through one of the old fashioned interface
```
Comment (VoyTechnology) [inline docs/plans/project-ideas.md]: As per other discussion on `*` vs `-` on subpoints

File: docs/plans/project-ideas.md [hunk 4]
```diff
@@ -1,66 +1,61 @@
...
+    forums, stuff that if you change, you may as well just move to real web forums, and leave INN2
+    running in the background. Also, isn't this primarly a webmaster thing? Anyway, just my 2c.- lil_cain
+  * [MuffinReader](https://github.com/Chewie/MuffinReader)
+* Motd system with database/web stuff so everyone can use it (receive)
+* Re-write rbusers, and hey in python/perl/ruby, and restore the admin/helpdesk categories that
+  used to exist. (receive)
+  * rbusers is done and, on hg.redbrick.dcu.ie (in python). If someone annoys me, I might even
+    package it - lil_cain
+  * I'm gonna work on this over the summer, and maybe add in some "hey helpdesk" that heys all
+    helpdesk members. If I do it, it'll be in python - creadak
```
Comment (VoyTechnology) [inline docs/plans/project-ideas.md]: `>` for both?

File: docs/procedures/mail_setup.md [hunk 5]
```diff
@@ -1,134 +1,140 @@
-Hi there, fresh-faced young redbrick admin. I expect you've noticed that since you became an admin you've been getting a lot more mail from members wanting accounts renewed/logwatch telling you the status of the machines/Nigerian princes offering you vast sums of money. This comes with the territory, unfortunately. The following are config files that will allow you to manage admin emails within mutt.
+# Mail Setup
 
-To make these configs work, you need to go to your ~/Maildir and run "mdmake `<mailbox_name>`", where mailbox_name is the
-name of each mailbox in the mailboxes line of the below .muttrc. e.g. "mdmake committee"
+Hi there, fresh-faced young redbrick admin. I expect you've noticed that since you became an admin
+you've been getting a lot more mail from members wanting accounts renewed/logwatch telling you the
+status of the machines/Nigerian princes offering you vast sums of money. This comes with the
```
Comment (VoyTechnology) [inline docs/procedures/mail_setup.md:5]: We are missing a section on hook-up requests

**Review by VoyTechnology — CHANGES_REQUESTED** [review]

File: docs/procedures/mail_setup.md (hunk 5 above)
Comment (butlerx) [inline docs/procedures/mail_setup.md]: ......what

File: docs/plans/project-ideas.md (hunk 2 above)
Comment (butlerx) [reply docs/plans/project-ideas.md]: can you have `>` in a list?

File: docs/plans/project-ideas.md (hunk 4 above)
Comment (butlerx) [reply docs/plans/project-ideas.md]: these dont mix well with `-`

**Review by VoyTechnology — APPROVED** [review]
LGTM

**Pull request merged**